src/libs/ReportUtils.js

```diff
@@ -514,6 +514,33 @@ function isExpensifyOnlyParticipantInReport(report) {
     return reportParticipants.length === 1 && _.some(reportParticipants, (accountID) => _.contains(CONST.EXPENSIFY_ACCOUNT_IDS, accountID));
 }
 
+/**
+ * Checks if the policy that the report is on is owned by one of the special Expensify accounts
+ *
+ * @param {Object} report
+ * @returns {Boolean}
+ */
+function doExpensifyAccountsOwnPolicy(report) {
```

Comment:
NAB but I feel like this is slightly clearer  (at least to me :D)
```suggestion
function isPolicyOwnedByExpensify(report) {
```

Also, I think this might be better placed in `actions/Policy.js` (the file already contains similar helper functions) and can just take a `policyID`, we already have `allPolicies` in that file as well. What do you think?

Reply:
A cleanup idea for later is to remove those helper functions from `actions/Policy.js` and into `PolicyUtils`

Reply:
Good points. Though actually, I'm going to refactor the `canCreateTaskInReport` function below, which means we'll no longer need this function.

Reply:
You're using it [here](https://github.com/Expensify/App/pull/30019/files#diff-65c096044d5f69b35bcdec14c99c4fda4580759df9b1a7c36650d58eea276f1dR3682) as well, are you refactoring that bit as well?

Reply:
That's the only place it'll be used, so no need to be a separate function is my thinking. Do you agree? I updated the PR so you can see how I'm doing it now.